cmd/podman/shared/create.go

```diff
@@ -804,7 +804,7 @@ func makeHealthCheckFromCli(c *GenericCLIResults) (*manifest.Schema2HealthConfig
 	if err != nil {
 		return nil, errors.Wrapf(err, "invalid healthcheck-timeout %s", inTimeout)
 	}
-	if timeoutDuration < time.Duration(time.Second*1) {
+	if timeoutDuration < time.Duration(1) {
```

Comment:
Does the linter accept (time.Duration(1) * time.Second)? I hate to lose the units here.